react/tic-tac-toe/src/redux/game/actions.js

```diff
@@ -20,6 +23,10 @@ const actionsCreators = {
       dispatch({ type: actions.MATCHES_FAILURE, payload: ERRORS[response.problem] });
       throw new SubmissionError({ _error: ERRORS[response.problem] });
     }
+  },
+  setGameSettings: value => dispatch => {
+    localStorageService.setValue(actions.GAME_SETTIGNS, { ...value });
```

Comment:
you could send just value, is the same as `{ ...value }`

Reply:
👆